user-setup.sh

```diff
@@ -10,21 +10,24 @@ set -xe
 # uid=33031(covidsimdev) gid=6493(PAS1694) groups=6493(PAS1694),6604(accesscovidsimdev),6602(PZS1008)
 
 groupadd -g 6697 PAS1788
-useradd -g PAS1788 -u 33929 covid
-useradd -g PAS1788 -u 33930 coviddev
-
 groupadd -g 6493 PAS1694
 groupadd -g 6602 PZS1008
 groupadd -g 6603 accesscovidsim
 groupadd -g 6604 accesscovidsimdev
+groupadd -g 7175 PDE0001
 
 useradd -g PAS1694 -u 33024 covidsim
 useradd -g PAS1694 -u 33031 covidsimdev
+useradd -g PDE0001 -u 36970 focalmap
```

Comment:
Same as other, good to make look like outside container:
```suggestion
useradd -g PZS0523 -G PDE0001 -u 36970 focalmap
```

Reply:
done.